### PR TITLE
keylime_tenant non-zero exit code on error

### PR DESCRIPTION
**Pull request by mpeters**
Now that we are catching errors and making them more user-friendly
we still need to exit with a non-zero exit code so that other
programs can tell when something went wrong

Signed-off-by: Michael Peters <mpeters@redhat.com>

### CODE REVIEW

**Review by THS-on — APPROVED** [review]

**Comment by kkaarreell** [comment]
I have pushed test update with exit codes, you can run `/packit test` again.

**Comment by mpeters** [comment]
Github is still having issues from an outage earlier today. I'll hold off on running the tests until actions are green again.

**Comment by kkaarreell** [comment]
I think this is expected, unless @sergio-correia has write access. I can only run tests for keylime PRs where I am the author.

**Comment by kkaarreell** [comment]
This is weird, for some reason c9s test were run against upstream code and not this PR.

**Comment by kkaarreell** [comment]
That's some tmt stuff, I will consult it with folks tomorrow. I think it is safe to ignore it for now.

**Comment by kkaarreell** [comment]
However the failure on c8s seems like a real issue

```
Reading configuration from ['/etc/keylime.conf']
Traceback (most recent call last):
  File "/usr/local/bin/keylime_tenant", line 11, in <module>
    load_entry_point('keylime==6.3.1', 'console_scripts', 'keylime_tenant')()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 476, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2700, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2318, in load
    return self.resolve()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.6/site-packages/keylime-6.3.1-py3.6.egg/keylime/cmd/tenant.py", line 9, in <module>
    from keylime import tenant
  File "/usr/local/lib/python3.6/site-packages/keylime-6.3.1-py3.6.egg/keylime/tenant.py", line 24, in <module>
    from keylime.agentstates import AgentAttestState
  File "/usr/local/lib/python3.6/site-packages/keylime-6.3.1-py3.6.egg/keylime/agentstates.py", line 9, in <module>
    from keylime.ima_ast import get_FF_HASH, get_START_HASH
  File "/usr/local/lib/python3.6/site-packages/keylime-6.3.1-py3.6.egg/keylime/ima_ast.py", line 21, in <module>
    from keylime.failure import Failure, Component
  File "/usr/local/lib/python3.6/site-packages/keylime-6.3.1-py3.6.egg/keylime/failure.py", line 197, in <module>
    SEVERITY_RULES, MAX_SEVERITY_LABEL = _eval_severity_config()
  File "/usr/local/lib/python3.6/site-packages/keylime-6.3.1-py3.6.egg/keylime/failure.py", line 181, in _eval_severity_config
    def rule(policy_regex: re.Pattern[Any], label_str: str, event_id: str) -> Optional[SeverityLabel]:
AttributeError: module 're' has no attribute 'Pattern'
```

**Review by maugustosilva — APPROVED** [review]

**Pull request merged**
